Project.toml

```diff
@@ -62,17 +62,17 @@ DiffEqCallbacks = "2.25"
 Downloads = "1.6"
 EllipsisNotation = "1.0"
 FillArrays = "0.13.2, 1"
-ForwardDiff = "0.10.18"
-HDF5 = "0.14, 0.15, 0.16, 0.17"
+ForwardDiff = "0.10.24"
+HDF5 = "0.17"
 IfElse = "0.1"
 LinearAlgebra = "1"
 LinearMaps = "2.7, 3.0"
-LoopVectorization = "0.12.118"
+LoopVectorization = "0.12.151"
 MPI = "0.20"
-Makie = "0.19, 0.20"
+Makie = "0.19"
```

Comment:
This seems wrong - don't we support the current Makie version?

Reply:
Yes, this was a mistake. I started with the compats from #1771 and forgot to update them. Should be corrected now.

Reply:
Thanks!